src/main/java/vivatech/init/VivatechEntities.java

```diff
@@ -2,8 +2,18 @@
 
 import net.minecraft.block.entity.BlockEntityType;
 import net.minecraft.util.registry.Registry;
-import vivatech.block.*;
-import vivatech.entity.*;
+import vivatech.block.CoalGeneratorBlock;
+import vivatech.block.CrusherBlock;
+import vivatech.block.ElectricFurnaceBlock;
+import vivatech.block.EnergyBankBlock;
+import vivatech.block.EnergyConduitBlock;
+import vivatech.block.PressBlock;
+import vivatech.entity.CoalGeneratorEntity;
+import vivatech.entity.CrusherEntity;
+import vivatech.entity.ElectricFurnaceEntity;
+import vivatech.entity.EnergyBankEntity;
+import vivatech.entity.EnergyConduitEntity;
+import vivatech.entity.PressEntity;
 
 public class VivatechEntities implements Initializable {
     public static final BlockEntityType<EnergyConduitEntity> ENERGY_CONDUIT;
```

Comment:
It should bind all tiered block to the entity. See below there’s a block instance given to the entity builder

Reply:
I'm not sure I understand. It binds all machine blocks of a given tier to the BlockEntity, isn't that correct?

Reply:
It should bind all of the ElectricFurnace blocks (all tiers) to the ElectricFurnaceEntity.
Like
`build(Electic...., ElFuBlock[0], ElFuBlock[1], ...`

Reply:
`...create(ElectricFurnaceEntity::new, VivatechBlocks.ELECTRIC_FURNACE[0], VivatechBlocks.ELECTRIC_FURNACE[1], VivatechBlocks.ELECTRIC_FURNACE[2])...`

Reply:
Oh right. It can also take an array, I think it functions the same. But if you'd prefer the entries split out I can change that.

Reply:
Ohhh, right, sry I forgot array is ok. Nothing to change :)